src/test/resources/tests.json

```diff
@@ -761,5 +761,65 @@
     },
     "input": "\n<pre><code>\nCode\n\n</code></pre>\n    ",
     "output": "```\n\nCode\n\n```"
+  },
```

Comment:
Could you add some tests here where the content inside the table has other htmls, also comments, and things like that?